editor/src/components/canvas/design-panel-root.tsx

```diff
@@ -13,7 +13,6 @@ import {
 import { useEditorState } from '../editor/store/store-hook'
 import { InspectorEntryPoint } from '../inspector/inspector'
 import { CanvasWrapperComponent } from './canvas-wrapper-component'
-import { InsertMenuPane } from '../navigator/left-pane'
```

Comment:
ultra nitpick, just FYI: we try to avoid changes to unrelated files in a PR, to reduce noise for the reviewers.

Reply:
The import statement has changed

Reply:
(I thought the same, that's why I knew the answer straight away :D)

Reply:
ooooh, sorry! I thought it's a leftover Organize Imports. sorry for the noise then!!